docs/constitution/pyth-dao-constitution.md

```diff
@@ -65,6 +66,8 @@ A PIP is submitted through a structured process via the **Pyth Forum** (e.g. Git
 - Key Terms - technical and/or commercial associated with the PIP
 - Implementation Plan - steps envisioned to implement the PIP, including resources needed for each step and timelines. The implementation plan may include binding on-chain actions that will automatically execute when the PIP passes.
 
+A council member uses the Operations Wallet to submit the proposed PIP on-chain.
```

Comment:
No this is not true the Operations wallet is only used to submit PIP to that council will vote on.